README.md

```diff
@@ -1,43 +1,115 @@
 # Cloud Prototyping 2019/2020
 
-## Motivation
 
-This repository was created to automate the benchmarking process of ethereum with caliper. We have created the following docker images and automation scripts to make the process portable and repeatable as possible.
 
-## Docker
+#A benchmarking-based approach to maximize throughput of the private Ethereum
 
-we import from [ethereum/client-go:alltools-stable](https://hub.docker.com/r/ethereum/client-go)  as it has most of the tools we need.
 
-So far we have three images:
+Our goals were to achieve :
+1) Performance Benchmarking. Benchmarking the performance of private Ethereum with Proof-of-Authority consensus  while maximizing Throughput depending on Block size and Block interval. 
```

Comment:
Some words have double spacing. Throughput should not be capital as it does not start in the beginning of the sentence.